cypress/e2e/tests/migration/applicationinventory/assessment/interlinked.test.ts

```diff
@@ -132,27 +119,35 @@ describe(["@tier3"], "Applications interlinked to tags and business service", ()
         const application = new Assessment(appdata);
         application.create();
         cy.get("@getApplication");
-        cy.wait(2000);
+        cy.wait(2 * SEC);
         // Perform assessment of application
         application.perform_assessment(
             "low",
             [stakeholdersList[0].name],
-            [stakeholdergroupsList[0].name]
+            [stakeholderGroupsList[0].name]
         );
         application.verifyStatus("assessment", "Completed");
 
         // Delete the stakeholders, group
         stakeholdersList[0].delete();
-        stakeholdergroupsList[0].delete();
+        stakeholderGroupsList[0].delete();
 
         clickByText(navMenu, applicationInventory);
         application.selectApplication();
         application.click_assess_button();
         click(continueButton);
-        cy.wait(6000);
+        cy.wait(6 * SEC);
 
         //Verify that values show blank
         cy.get(stakeholderSelect).should("have.value", "");
         cy.get(stakeholdergroupsSelect).should("have.value", "");
     });
+
+    after("Perform test data clean up", function () {
+        deleteAllStakeholders();
```

Comment:
Let's delete data created in this test only not all .
That interferes with other tests.

Reply:
Done